tests/subsys/usb/device_next/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 common:
-  depends_on: usb_device
+  depends_on: usbd
   platform_allow:
     - native_sim
     - qemu_cortex_m3
```

Comment:
what about this qemu? it is among platform_allow with depends_on: usbd but I don't see you adding such entry to the board in this PR. If it is not supported then I think it has to be taken out from platform_allow here and in uac2 sample, preferably replaced with some other qemu

Reply:
Just no value to do it and I am not interested in running it elsewhere yet.